### PR TITLE
[DOCS] Documents that embeddings are filtered out from `semantic_text` results 

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

When displayed in an API response for semantic_text fields, the embeddings are automatically filtered out from `_source`. This PR changes the API results in the documentation and documents the `include_vectors` parameter in `_source` documentation.

### Preview

* [semantic_text tutorial](https://elasticsearch_bk_113701.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/semantic-search-semantic-text.html)
* [_source](https://elasticsearch_bk_113701.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/mapping-source-field.html#filter-vectors)

### CODE REVIEW

File: docs/reference/mapping/fields/source-field.asciidoc [hunk 1]
```diff
@@ -122,3 +122,96 @@ GET logs/_search
...
+{
+  "_source": { "include_vectors": true },
+  "query": {
+    "match": {
+      "content": "Test Data"
+    }
+  }
+}
+--------------------------------------------------
+
```
Comment (szabosteve) [inline docs/reference/mapping/fields/source-field.asciidoc:153]:
```suggestion
// TEST[skip: TBD]

```

**Comment by szabosteve** [comment]
@elasticsearchmachine update branch

File: docs/reference/mapping/fields/source-field.asciidoc [hunk 2]
```diff
@@ -122,3 +122,97 @@ GET logs/_search
...
+The default behavior varies by field typse:
+
+* `sparse_vector` and `dense_vector` fields are included by default if `include_vectors` is not set to `false`
+* `semantic_text` fields are excluded by default if `include_vectors` is not set to `true`
+  * `update_by_query` and `reindex` operations on `semantic_text` fields are included by default if `include_vectors` is not set to `false`
+
+
+The `include_vectors` parameter can be used as follows:
+
+[source,console-result]
```
Comment (szabosteve) [inline docs/reference/mapping/fields/source-field.asciidoc:141]:
```suggestion
[source,console]
```

**Comment by Mikep86** [comment]
@szabosteve We can close this PR. we are no longer going to implement this filtering, instead we are going to change how semantic text field values are stored in source.

**Comment by szabosteve** [comment]
@Mikep86 Thanks for the heads-up! I'm closing this.

**Pull request closed without merging**